packages/airnode-node/src/coordinator/calls/disaggregation.test.ts

```diff
@@ -75,7 +77,8 @@ describe('disaggregate - Requests', () => {
     const aggregatedApiCallsById = { btcCall: aggregatedApiCall };
 
     const config = fixtures.buildConfig();
-    const state = coordinatorState.create(config);
+    const coordinatorId = randomHexString(16);
```

Comment:
Why move the `coordinatorId` from the `create` function?

Reply:
I moved it so that we can use `setLogOptions` at the start of the `startCoordinator` handler instead of having to do it inside `createInitialCoordinatorState`. I thought this seemed neater to match the other handlers, but can revert if you don't like it :smile:

Reply:
Oh, I see. This `coordinatorId` also complicates the suggestion:

```
It feels to me the logger should be initialized by deployer handlers and [when the Airnode client is invoked](https://github.com/api3dao/airnode/blob/master/packages/airnode-node/src/cli/invoke.ts#L7)
```

Reply:
I wonder if it would be worth it do define CLI specific logger-methods (similar to what airnode-examples has with `cliPrint`).

Reply:
Yeah. Another thing is that we are mixing "generic purpose logging" with "contextual Airnode logging" (using the pending logs, fixed coordinator ID set at the beginning) which could be split as well.